src/passenger.ts

```diff
@@ -20,9 +20,13 @@ const passenger: CrimboQuest = {
         return chooseQuestOutfit({ location, isFree: false }, drunkSpec);
       },
       effects: () =>
-        $effects`Blood Bubble, Blood Bond, Frenzied\, Bloody, Empathy, Leash of Linguini, Ruthlessly Efficient, Mathematically Precise`.filter(
-          (effect) => have(toSkill(effect))
-        ),
+        [
+          ...$effects`Blood Bubble, Blood Bond, Frenzied\, Bloody, Empathy, Leash of Linguini, Ruthlessly Efficient, Mathematically Precise, Psalm of Pointiness`,
+          toEffect($skill`Shield of the Pastalord`),
```

Comment:
i think this is good to have but my one small callout is that this is pretty MP regen heavy and we probably need a few things that restore MP in our outfit to use this. i think it is very worth adding tho.